src/metaswitch/clearwater/config_manager/etcd_synchronizer.py

```diff
@@ -62,7 +63,10 @@ def main(self):
                     self._plugin.file(),
                     len(value),
                     md5(value.encode("utf-8", errors="replace")).hexdigest()))
```

Comment:
It would be nice if you used your new function here as well. If you do, you should rename the function as it's no longer just for logging (it's more about safely encoding a string)